src/libs/ModifiedExpenseMessage.ts

```diff
@@ -188,16 +186,31 @@ function getForReportAction(reportID: string | undefined, reportAction: OnyxEntr
 
     const hasModifiedTag = reportActionOriginalMessage && 'oldTag' in reportActionOriginalMessage && 'tag' in reportActionOriginalMessage;
     if (hasModifiedTag) {
-        buildMessageFragmentForValue(
-            PolicyUtils.getCleanedTagName(reportActionOriginalMessage?.tag ?? ''),
-            PolicyUtils.getCleanedTagName(reportActionOriginalMessage?.oldTag ?? ''),
-            policyTagListName,
-            true,
-            setFragments,
-            removalFragments,
-            changeFragments,
-            policyTagListName === Localize.translateLocal('common.tag'),
-        );
+        const policyTags = allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`] ?? {};
+        const transactionTag = reportActionOriginalMessage?.tag ?? '';
+        const oldTransactionTag = reportActionOriginalMessage?.oldTag ?? '';
+        const splittedTag = transactionTag.split(CONST.COLON);
+        const splittedOldTag = oldTransactionTag.split(CONST.COLON);
```

Comment:
Not a huge deal, but it would be good to add a utility function for this?

Reply:
Do we really create such helpers? If yes, where to put this, because it is being used across the project?

Reply:
Fixed this a bit here: https://github.com/Expensify/App/pull/36978